hw/ip/edn/doc/theory_of_operation.md

```diff
@@ -71,9 +71,10 @@ For details on the options for application interface commands please see the [CS
 Once the CSRNG instance has been instantiated, and the `generate` and `reseed` commands have been loaded, auto request mode can be entered by programming the [`CTRL`](registers.md#ctrl) register with `EDN_ENABLE` and `AUTO_REQ_MODE` fields are enabled.
 Note that if BOOT_REQ_MODE is asserted the state machine will enter boot-time request mode, even if AUTO_REQ_MODE is asserted.
 
-To issue any new commands other than those stored in the generate or reseed FIFOs, it is important to disable auto request mode, by deasserting the `AUTO_REQ_MODE` field in the [`CTRL`](registers.md#ctrl) register.
+To issue any new commands other than those stored in the generate or reseed FIFOs, it is important to disable auto request mode, by deasserting the `AUTO_REQ_MODE` or the `EDN_ENABLE` field in the [`CTRL`](registers.md#ctrl) register.
 Firmware must then wait until the current command is completed by polling the [`MAIN_SM_STATE`](registers.md#main_sm_state) register.
 Once the state machine returns to the `Idle` or `SWPortMode` states, new firmware-driven commands can be passed to the CSRNG via the [`SW_CMD_REQ`](registers.md#sw_cmd_req) register.
+Any time the EDN is disabled, the `SWPortMode` state is entered or the boot sequence has completed, the generate and reseed FIFOs are reset.
```

Comment:
I think this might be easier to understand if the sentence gets split up a bit? I *think* I interpreted this right. Another possible meaning is that the FIFOs are also reset whenever the EDN gets disabled (in which case my suggested text below is wrong!)

```
The generate and reseed FIFOs are reset whenever the `SwPortMode` state is entered while the EDN is disabled.
They are also reset on completion of the boot sequence.
```

Reply:
Okay in that case it's probably best I change this. Thanks for reviewing.